dapps/digital-twin-detail/src/i18n/de.json

```diff
@@ -86,7 +86,7 @@
         "user": "Nutzer"
       },
       "no-sharings": "Dieser Digitale Zwilling wurde mit keinem Nutzer geteilt.",
-      "no-sharings-filter": "Dieser Digitale Zwilling wurde mit keinem Nutzer mit diesem Filter gefunden. (Suche: {searchTerm} und Filter: {filter})",
+      "no-sharings-filter": "Dieser Digitale Zwilling wurde mit keinem Nutzer geteilt der zu \"{searchTerm}\" und Typ: \"{filter}\" passt.",
```

Comment:
Did we actually discuss if we even need to display the searchTerm and Filter? It's redundant imo and makes the no-result message way longer than it needs to be.